oca_dependencies.txt

```diff
@@ -1,3 +1,3 @@
 # List the OCA project dependencies, one per line
 # Add a repository url and branch if you need a forked version
-interface-github
+interface-github https://github.com/grap/interface-github 10.0_ADD_github_connector_github_connector_odoo
```

Comment:
The github_connector module is now merged, to this can be reverted back to the original oca/interface-github repo

Reply:
No, it is not yet merged into 10.0
https://github.com/OCA/apps-store

Reply:
For OCA it is: https://github.com/OCA/interface-github

Reply:
ohh 0kay.  changes are committed